src/common/stringUtils.ts

```diff
@@ -44,7 +44,7 @@ export function graphemePositionFrom(
     text: string,
     lineCodeUnit: number,
     lineNumber: number,
-    maybeCodeUnit: number | undefined,
```

Comment:
Are these suppose to have trailing commas as well?

Reply:
I thought so, but I don't fully understand the prettier config.

Reply:
For now I've set trailingComma to always.